README.md

```diff
@@ -1,7 +1,45 @@
-# COMPASS
+## COMPASS
+
 COregistered Multi-temPorAl Sar Slc
 
-# License
+🚨 This toolbox is still in **pre-alpha** stage and undergoing **rapid development**. 🚨
+
+### Install
+
+
+The following will install COMPASS into a conda environment.
+
+
```

Comment:
I have ignored the other codacy style issue on their website. The change below should fix the remaining codacy warnings.

```suggestion

The following will install COMPASS into a conda environment.

```